resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosSchedulerUtils.scala

```diff
@@ -405,6 +405,20 @@ trait MesosSchedulerUtils extends Logging {
       sc.executorMemory
   }
 
+  /**
+   * Return the amount of memory to allocate to each driver, taking into account
+   * container overheads.
+   *
+   * @param driverDesc used to get driver memory
+   * @return memory requirement as (0.1 * memoryOverhead) or MEMORY_OVERHEAD_MINIMUM
```

Comment:
Looks like the actual description here is ->
1. DRIVER_MEMORY_OVERHEAD if set
2. max of MEMORY_OVERHEAD_MINIMUM or 0.1 * driver memory

Reply:
Yes, that's right.  Updated this comment.